sale_commission/models/account_invoice.py

```diff
@@ -128,13 +123,13 @@ def onchange_agent(self):
     @api.depends('commission.commission_type', 'invoice_line.price_subtotal')
```

Comment:
Now that I'm revisiting the code, I think you should remove `commission.commission_type` from the depends section, because if you change the type in the future, old past invoices will be recalculated.